rotkehlchen/ethchain.py

```diff
@@ -76,6 +102,19 @@ def set_rpc_port(self, port: int) -> Tuple[bool, str]:
             self.ethrpc_port = port
         return result, message
 
+    def get_eth_highest_block(self):
```

Comment:
add the return type of `int` here too and please change the name of the function to signify that this is querying an external source for the highest block.

Something like `query_eth_highest_block()`